graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -137,19 +138,21 @@
 VisitorReturnType = Union[Node, VisitorAction]
 
 
-class RenamingMapping(Protocol):
+class TypeRenamingMapping(Protocol):
     def get(self, key: str, default: Optional[str]) -> Optional[str]:
-        """Define mapping for renaming object."""
+        """Define mapping for type_renamings object."""
         ...
 
 
-def rename_schema(schema_ast: DocumentNode, renamings: RenamingMapping) -> RenamedSchemaDescriptor:
-    """Create a RenamedSchemaDescriptor; rename/suppress types and root type fields using renamings.
+def rename_schema(
+    schema_ast: DocumentNode, type_renamings: TypeRenamingMapping
+) -> RenamedSchemaDescriptor:
+    """Create a RenamedSchemaDescriptor; rename/suppress types and root type fields.
 
-    Any type, interface, enum, or fields of the root type/query type whose name
-    appears in renamings will be renamed to the corresponding value if the value is not None. If the
-    value is None, it will be suppressed in the renamed schema and queries will not be able to
-    access it.
+    Any object type, interface type, enum type, or field of the root type/query type has a name. Let
+    the name be called type_name. If type_renamings.get(type_name, type_name) is not None, the type
```

Comment:
962 comment: 

Needed to revise this `type_renamings` comment because the original comment mentions the idea of a name being "in" `type_renamings`, which only applies if `type_renamings` is iterable. If, instead, we have an object that simply implements a get method that maps strings to strings, it doesn't make sense to speak of anything being "in" the object. 

There was some discussion previously about whether the original paragraph was clearer versus this new explanation, but I find that the new one is more accurate so I kept this change here.